### PR TITLE
Update Data Integration Tutorial

### PR DESCRIPTION
**Pull request by DMadhuranga**
Change the data integration tutorial according to the new DSS editor.

### CODE REVIEW

File: en/micro-integrator/docs/use-cases/tutorials/sending-a-simple-message-to-a-datasource.md [hunk 1]
```diff
@@ -2,15 +2,16 @@
 
 ## What you'll build
 
-A **data service** provides a Web service interface to access data that is stored in relational databases, CSV files, Microsoft Excel sheets,
-Google spreadsheets, and more. The following sections describe how you can use WSO2 Integration Studio to work with data services' artifacts. 
+A **data service** provides a web service interface to access data that is stored in various datasources. The following sections describe how you can use WSO2 Integration Studio to work with data services' artifacts.
```
Comment (nilminiwso2) [inline en/micro-integrator/docs/use-cases/tutorials/sending-a-simple-message-to-a-datasource.md]: @DMadhuranga @rosensilva - Hope this description is acceptable.

File: en/micro-integrator/docs/use-cases/tutorials/sending-a-simple-message-to-a-datasource.md [hunk 2]
```diff
@@ -85,9 +87,19 @@ Follow the steps given below to create the data service file:
...
-4.  Click **Next** and start adding the datasource connection details
-    given below.
+![](../../assets/img/tutorials/data_services/dataservice_view.png)
+
+##### Creating the datasource connection
+
+1.  Click **Data Sources** to expand the section.
+    ![](../../assets/img/tutorials/data_services/data_source_expanded.png)
+2.  Click **Add New** to open the **Create Datasource** page.
+    ![](../../assets/img/tutorials/data_services/add_data_source.png)
```
Comment (nilminiwso2) [inline en/micro-integrator/docs/use-cases/tutorials/sending-a-simple-message-to-a-datasource.md]: @DMadhuranga - Shall we change this to an empty screenshot? The descriptions are more meaningful that way.

File: en/micro-integrator/docs/use-cases/tutorials/sending-a-simple-message-to-a-datasource.md [hunk 3]
```diff
@@ -99,52 +111,46 @@ Follow the steps given below to create the data service file:
     | URL                                | jdbc:mysql://localhost:3306/Employees |
     | User Name                          | root                                  |
 
-5.  Save the data service.
+4.  Click **Test Connection** to expand the section.
+    ![](../../assets/img/tutorials/data_services/test_connection.png)
```
Comment (nilminiwso2) [inline en/micro-integrator/docs/use-cases/tutorials/sending-a-simple-message-to-a-datasource.md]: @DMadhuranga - Shall we add a screenshot for this? Same as for 'datasource connection details'.
Let's keep this pattern of screenshots consistent.

File: en/micro-integrator/docs/use-cases/tutorials/sending-a-simple-message-to-a-datasource.md [hunk 4]
```diff
@@ -99,52 +111,46 @@ Follow the steps given below to create the data service file:
...
+##### Creating a query
 
 Let's write an SQL query to GET data from the MySQL datasource that you
 configured in the previous step:
 
-1.  Select the data service you created in the previous step.
-2.  Right-click and click **Add Query** .  
-    ![](../../assets/img/tutorials/data_services/119130577/119130591.png)
+1.  Click **Queries** to expand the section. 
+    ![](../../assets/img/tutorials/data_services/data_source_expanded.png)
```
Comment (nilminiwso2) [inline en/micro-integrator/docs/use-cases/tutorials/sending-a-simple-message-to-a-datasource.md]: @DMadhuranga - Shall we add a screenshot for this? Same as for 'datasource connection details'.
Let's keep this pattern of screenshots consistent.

File: en/micro-integrator/docs/use-cases/tutorials/sending-a-simple-message-to-a-datasource.md [hunk 5]
```diff
@@ -99,52 +111,46 @@ Follow the steps given below to create the data service file:
...
+2.  Click **Add New** to open the **Add Query** page.
+    ![](../../assets/img/tutorials/data_services/add_query.png)
```
Comment (nilminiwso2) [inline en/micro-integrator/docs/use-cases/tutorials/sending-a-simple-message-to-a-datasource.md]: @DMadhuranga - Shall we add a screenshot for this? Same as for 'datasource connection details'.
Let's keep this pattern of screenshots consistent.

File: en/micro-integrator/docs/use-cases/tutorials/sending-a-simple-message-to-a-datasource.md [hunk 6]
```diff
@@ -99,52 +111,46 @@ Follow the steps given below to create the data service file:
...
-    ```bash
-    select EmployeeNumber, FirstName, LastName, Email from Employees where EmployeeNumber=:EmployeeNumber
-    ```
-
-6.  Save the SQL statement.
-7.  Right-click the query again and click **Add Input Mapping** .
-
-8.  Enter the following input mapping details:
+4.  Click **Input Mappings** to expand the section. 
+    ![](../../assets/img/tutorials/data_services/data_source_expanded.png)
```
Comment (nilminiwso2) [inline en/micro-integrator/docs/use-cases/tutorials/sending-a-simple-message-to-a-datasource.md]: @DMadhuranga - Shall we add a screenshot for this? Same as for 'datasource connection details'.
Let's keep this pattern of screenshots consistent.

File: en/micro-integrator/docs/use-cases/tutorials/sending-a-simple-message-to-a-datasource.md [hunk 7]
```diff
@@ -99,52 +111,46 @@ Follow the steps given below to create the data service file:
...
+5.  Click **Add New** to open the **Add Input Mapping** page.
+    ![](../../assets/img/tutorials/data_services/add_input_mappings.png)
```
Comment (nilminiwso2) [inline en/micro-integrator/docs/use-cases/tutorials/sending-a-simple-message-to-a-datasource.md]: @DMadhuranga - Shall we add a screenshot for this? Same as for 'datasource connection details'.
Let's keep this pattern of screenshots consistent.

File: en/micro-integrator/docs/use-cases/tutorials/sending-a-simple-message-to-a-datasource.md [hunk 8]
```diff
@@ -99,52 +111,46 @@ Follow the steps given below to create the data service file:
...
     | Parameter Type | SCALAR         |
     | SQL Type       | STRING         |
 
-9.  Save the input mapping.
-10. Right-click the query again and click **Add Output Mapping**.
-11. Enter the following value to group the output mapping:
+5.  Save the input mapping.
+    ![](../../assets/img/tutorials/data_services/input_mappings.png)
+6.  Click **Result (Output Mappings)** to expand the section.
+    ![](../../assets/img/tutorials/data_services/data_source_expanded.png)
```
Comment (nilminiwso2) [inline en/micro-integrator/docs/use-cases/tutorials/sending-a-simple-message-to-a-datasource.md]: @DMadhuranga - Shall we add a screenshot for this? Same as for 'datasource connection details'.
Let's keep this pattern of screenshots consistent.

File: en/micro-integrator/docs/use-cases/tutorials/sending-a-simple-message-to-a-datasource.md [hunk 9]
```diff
@@ -157,12 +163,58 @@ configured in the previous step:
...
+
+##### Creating a resource to invoke the query
 
-13. Right-click the output mapping and go to **Add Output Mapping → Add
-    Element** to create an element.
+Now, let's create a REST resource that can be used to invoke the query.
 
-14. Enter the following element details.
+1.  Click **Resources** to expand the section. 
+    ![](../../assets/img/tutorials/data_services/data_source_expanded.png)
```
Comment (nilminiwso2) [inline en/micro-integrator/docs/use-cases/tutorials/sending-a-simple-message-to-a-datasource.md]: @DMadhuranga - Shall we add a screenshot for this? Same as for 'datasource connection details'.
Let's keep this pattern of screenshots consistent.

File: en/micro-integrator/docs/use-cases/tutorials/sending-a-simple-message-to-a-datasource.md [hunk 10]
```diff
@@ -157,12 +163,58 @@ configured in the previous step:
...
+2.  Click **Add New** to open the **Create Resource** page.
+    ![](../../assets/img/tutorials/data_services/create_resource.png)
```
Comment (nilminiwso2) [inline en/micro-integrator/docs/use-cases/tutorials/sending-a-simple-message-to-a-datasource.md]: @DMadhuranga - Shall we add a screenshot for this? Same as for 'datasource connection details'.
Let's keep this pattern of screenshots consistent.

File: en/micro-integrator/docs/use-cases/tutorials/sending-a-simple-message-to-a-datasource.md (hunk 9 above)
Comment (DMadhuranga) [reply en/micro-integrator/docs/use-cases/tutorials/sending-a-simple-message-to-a-datasource.md]: Done

File: en/micro-integrator/docs/use-cases/tutorials/sending-a-simple-message-to-a-datasource.md (hunk 4 above)
Comment (DMadhuranga) [reply en/micro-integrator/docs/use-cases/tutorials/sending-a-simple-message-to-a-datasource.md]: Done

**Pull request merged**